### PR TITLE
FAT-8657 Writing test coverage for EDGEDCB 

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
The Purpose of this pr is to add karate tests to cover edge dcb api calls.

## Approach
Added karate tests for edge-dcb module.

## Evidence
Tested using volaris rancher 
image

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: common/src/main/resources/common/setup-users.feature [hunk 1]
```diff
@@ -2,7 +2,7 @@ Feature: prepare data for api test
 
   Background:
     * url baseUrl
-    * configure readTimeout = 300000
+    * configure readTimeout = 3000000
```
Comment (gurleenkaurbp) [inline common/src/main/resources/common/setup-users.feature:5]: please revert

File: common/src/main/resources/common/tenant.feature [hunk 2]
```diff
@@ -3,6 +3,7 @@ Feature: Tenants
   Background:
     * url baseUrl
     * configure retry = { count: 2, interval: 5000 }
+    * configure readTimeout = 3000000
```
Comment (gurleenkaurbp) [inline common/src/main/resources/common/tenant.feature:7]: is this required otherwise please revert it

File: mod-dcb/src/main/resources/karate-config.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+    config.admin = {
+      tenant: 'supertenant',
+      name: 'testing_admin',
+      password: 'admin'
+    }
+  } else if (env == 'rancher') {
+   // config.baseUrl = 'https://folio-dev-volaris-okapi.ci.folio.org';
+   // config.edgeUrl = 'https://folio-dev-volaris-edge-dcb.ci.folio.org';
+
```
Comment (gurleenkaurbp) [inline mod-dcb/src/main/resources/karate-config.js:104]: Please remove commented lines of code.

File: mod-dcb/src/main/resources/volaris/mod-dcb/features/util/initData2.feature [hunk 4]
```diff
@@ -0,0 +1,172 @@
+Feature: init data for mod-dcb
```
Comment (gurleenkaurbp) [inline mod-dcb/src/main/resources/volaris/mod-dcb/features/util/initData2.feature:1]: Why 2 initData feature files ?

File: mod-dcb/src/main/resources/volaris/mod-dcb/features/util/initData3.feature [hunk 5]
```diff
@@ -0,0 +1,90 @@
+Feature: init data for mod-dcb
+
```
Comment (gurleenkaurbp) [inline mod-dcb/src/main/resources/volaris/mod-dcb/features/util/initData3.feature:2]: This is third initData feature file ? But there should not be more than 1 initData feature files
